docs/Usage-Guide.md

```diff
@@ -1,3 +1,172 @@
-# Usage Guide
-
----Organize the Usage Guide into sections using Markdown Headings. This guide should mirror the steps in the Quick Start but with far more details and include any optional steps.---
+# Usage Guide
+
+1. Download the package from the repository [Releases](https://github.com/Kentico/xperience-by-kentico-sitefinity-migration-tool/releases)
+
+   - Optionally, clone the git repository and build from the source files.
+   - ` git clone -v "https://github.com/Kentico/xperience-by-kentico-sitecore-migration-tool.git" "{YOUR PATH}"`
+
+2. Modify the console application's configuration file `/examples/Migration.Toolkit.Sitefinity.Console/appsettings.json`
+   - The details for configuration options can be found below.
+3. Modify code for any additional customizations
+   - Entity Framework is included. Add additional custom models to Migration.Toolkit.Sitefinity.Data.Core.EF.SitefinityContext for any custom tables. An additional import service will need to be created. See other import services.
+   - Use the RestSdkBase when creating a provider that calls the rest api endpoints. This will ensure the rest client is running and provides methods to call the rest client.
+   - Add additional field types for custom field types. See '/src/Migration.Toolkit.Sitefinity/FieldTypes' for OOB field types in Sitefinity. Using the IFieldType, will automatically add it to the import processing. You can also use the FieldTypeBase to handle the defaults.
+4. Set up Sitefinity
+   - Ensure 'Deployment Mode' is set to 'source'. This feature will export the content type definitions from the database and store them on the disk. Like XbyK's Continuous Integration (CI) files. ![alt text](image-5.png)

```

Comment:
If you end a line with two spaces, a newline is inserted. Just a tip.

Reply:
Good to know thank you!